diwan/src/commands/file_commands.rs

```diff
@@ -0,0 +1,24 @@
+pub enum CommandType {
+    Open,
+    Save,
+    SaveAs,
```

Comment:
if we are going to adopt this enum then please don't forget to handle `SaveAs` by add its method in the `impl`